ui/src/routes/_layout/organizations/$orgId/products/$productId/-components/last-run-id.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import { Loader2 } from 'lucide-react';
+
+import { useRepositoriesServiceGetOrtRunsByRepositoryId } from '@/api/queries';
+import { config } from '@/config';
+
+export const LastRunId = ({ repoId }: { repoId: number }) => {
```

Comment:
Why is this called LastRunId? Would something like TotalRunsForRepository or something be more descriptive of what this component returns?

Reply:
Oh, nice catch - it was LastRunId because I had initially the idea to just return the last (local) run ID. Then I realized that a far more useful is to return `totalCount` but forgot to change the name of the component :smile: